packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -572,22 +599,33 @@ export class LitContracts {
   };
 
   public static async getStakingContract(
-    network: 'cayenne' | 'manzano' | 'habanero' | 'custom' | 'localhost',
+    network:
+      | 'cayenne'
+      | 'manzano'
+      | 'habanero'
+      | 'custom'
+      | 'localhost'
+      | 'datil-dev',
     context?: LitContractContext | LitContractResolverContext,
     rpcUrl?: string
   ) {
     let provider: ethers.providers.JsonRpcProvider;
-    rpcUrl = rpcUrl ?? DEFAULT_READ_RPC;
+    rpcUrl = rpcUrl ?? LIT_RPC.VESUVIUS;
```

Comment:
As above, this feels like a breaking chain; I think we can't change this without breaking current users touching existing networks and need to make this default based on `network` if it's not passed to be backwards compatible